Rectangle/Rectangle/RectangleStruct.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RectangleStruct
+{
+    public struct Rectangle
+    {
+        private double width;
+        private double height;
+
+        public double Width
+        {
+            get => width;
+            set
+            {
+                Console.OutputEncoding = System.Text.Encoding.UTF8;
+                if (value <= 0)
+                    throw new ArgumentException("Ширина должна быть положительным числом");
+                width = value;
+            }
+        }
+
+        public double Height
+        {
+            get => height;
+            set
+            {
+                Console.OutputEncoding = System.Text.Encoding.UTF8;
+                if (value <= 0)
```

Comment:
Тип данных не должен быть привязан к консоли
```suggestion
            {
                if (value <= 0)
```